data/i18n/en/en.toml

```diff
@@ -519,12 +519,27 @@ other = "2006-01-02"
 [release_date_format_month]
 other = "January 2006"
 
+# Use 2006-01-02 (ISO 8601) if you are not sure what to use.
+[release_date_format_text]
+other = "Monday, 02 Jan 2006"
```

Comment:
I don't think this will work for localization teams.
This encoding is Golang specific, and the convention here won't translate well into any other languages.

ISO 8601 is a standard that can be followed by all localization teams though.

Reply:
> # Use 2006-01-02 (ISO 8601) if you are not sure what to use.

I could change that to explicitly mention that Hugo doesn't support localized date formatting. For English, we can still provide the tooltip, and if Hugo gains support, this will Just Work for other locales.

How about that?

Reply:
Looks like we can localize dates, but we have to use the exact language codes.
https://discourse.gohugo.io/t/localizing-date-format/37868 has more on this.